charter.md

```diff
@@ -0,0 +1,22 @@
+# Open Web Docs Charter
+
+## Mission
+
+Open Web Docs supports web platform documentation for the benefit of web developers & designers worldwide.
+
+
+## Core Values
+
+Open Web Docs believes open web platform documentation is critical digital infrastructure that benefits everyone. To demonstrate that belief, we: 
+
+- _focus_ on documentation that has a positive impact on end-users and web developers.
+- _prioritize_ based on needs of the global community of web developers and designers. 
+- _maintain_ an independent editorial voice, with a “firewall” between fiscal and editorial matters. 
+- _adhere_ to open source principles of transparency in project governance, investment, and feature development in order to maintain trustworthiness and credibility with the web community. 
+- _work_ to make the platform more equitable, investing in documenting internationalization, accessibility, privacy and diversity components of the web. 
+- _emphasize_ diversity and inclusion across all our efforts, including project contributions, community organization, and resource allocation.
+
+## What We Do
+To execute our mission and values, we contribute to MDN and other related efforts with an independent editorial voice, rooted in the web technology landscape reflecting the lived experience of web developers. We take a lead role on specific content projects within these platforms, in coordination with these platforms where appropriate.
+
+We invest in and protect Open Web Documentation as a critical piece of infrastructure which other organizations can build upon and are incentivized to protect. We ensure developers have the best information available to make technology and design decisions and to implement on the web platform.
```

Comment:
This reads slightly awkwardly, and is somewhat repetitious ("critical digital infrastructure" also mentioned in the "Core Values" section).

How about this:

"Open Web Docs works hard both to invest in and protect documentation for the open web platform, and to provide incentives and a platform for other related organizations to get involved in this work. We ensure developers have the best information available to make technology and design decisions and implement on the web platform."

Reply:
Thanks, updated! I left out the "works hard" part of this, though :)